lib/Raven/Legacy.php

```diff
@@ -14,7 +16,7 @@ class Raven_Client extends \Raven\Client
 {
 }
 
-/**
+/** @noinspection PhpDeprecationInspection
```

Comment:
Given that I was not aware of this tag (thank you for letting me know it 😄), from what I understand it means that IDEs like PHPStorm should not alert about the deprecation of this class. But if the class is deprecated there is no point to not signal it to the users. At least, this is my opinion...